types/jsdom-simulant/index.d.ts

```diff
@@ -0,0 +1,175 @@
+// Type definitions for jsdom-simulant 1.1
+// Project: https://github.com/https://github.com/lukebayes/simulant/
+// Definitions by: Dolan Murvihill <https://github.com/dmurvihill>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+export as namespace simulant;
```

Comment:
It seems like this is a callable module, so you should follow [this template](http://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-function-d-ts.html).